releasenotes/releasenotes.rst

```diff
@@ -24,11 +24,56 @@ Public Endpoint Announcement 8 July 2021
 
 The default FHIR version of the `public Firely Server endpoint <https://server.fire.ly/>`_ is now R4.
 
+
+.. _vonk_releasenotes_460:
+
+Release 4.6.0
+-------------
+
+Database
+^^^^^^^^
+
+#. SQL Server
+
+   #. A new computed column IsDeleted on table [vonk].[entry] is leveraged for more performant SQL queries (only applicable when ``Vonk.Repository.Sql.Raw.KSearchConfiguration`` plugin is enabled)
+   #. Improved performance of some SQL queries by avoiding unnecessary SQL query parameter type conversion (only applicable when ``Vonk.Repository.Sql.Raw.KSearchConfiguration`` plugin is enabled)
```

Comment:
Improved performance of SQL queries by converting 5 columns from vonk.entry to varchar upon retrieval: InformationModel, Type, ResourceId, Version, Reference. 
note: These columns should - by definition of the FHIR datatypes - not contain characters outside the varchar range, but please pay attention to this change if your id's or custom resourcetype has those characters nonetheless. We may alter the datatype of the columns in a future release.

Reply:
Improved performance of some SQL queries by avoiding excessive retrieval of the (large) ResourceJson column.